project/app/src/main/java/com/fillr/example/integration/activity/ExampleWebViewHeadlessActivity.java

```diff
@@ -93,11 +119,14 @@ public void onClick(DialogInterface dialog, int which) {
      * @see <a href="https://developer.fillr.com">Developer Center</a>
      */
     private void setupFillr(WebView webView) {

```

Comment:
So the Fillr setup repeated for each webview?

Reply:
yeah i thought about that; it's kept that way for simplicity.